hack/quickstart-setup.sh

```diff
@@ -0,0 +1,362 @@
+#!/bin/bash
+
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+BIN_DIR=$(pwd)/bin
+
+KIND_VERSION=v0.17.0
+
+export KIND_BIN=kind
+export YQ_BIN=yq
+
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.kindUtils)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.clusterUtils)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.cleanupUtils)"
+# TODO: Uncomment when otherwise ready to merge
+# source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.dependenciesUtils.sh)"
+source ./.dependenciesUtils.sh
```

Comment:
For testing can we make this the below as I'm running it outside of the MGC repo in order to ensure we're not getting any local dependencies

```suggestion
# TODO: Update to Kuadrant main before merging.
source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/laurafitzgerald/multicluster-gateway-controller/quick-start/hack/.dependenciesUtils.sh)"
```